.github/DEVELOPMENT.md

```diff
@@ -171,202 +89,6 @@ These are tests that will not run on a device. This is useful for testing device
-
-3. Open `obj\Debug\net[current_sdk_version]-android\Resource.designer.cs`, and find the
-   field you need such as:
-
-```csharp
-// aapt resource value: 0x7F010000
-public static int foo = 2130771968;
-```
-
-4. Copy this field to the `Resource.designer.cs` checked into source
-   control, such as: `src\Controls\src\Core\Platform\Android\Resource.designer.cs`
-
-5. Restore the commented code in `Directory.Build.targets`.
-
-
-# Advanced Scenarios
-
-### Compile using a local `bin\dotnet` via `dotnet-local.*`
-
-This method will use the .NET and workload versions that are specific to the branch you are on, which is a good way to ensure compatibility.
-
-Use `dotnet-local.cmd` on Windows or `dotnet-local.sh` on Unix to ensure that `PATH` is set consistently.
-
-#### Cake
-
-You can run a `Cake` target to bootstrap .NET SDK in `bin\dotnet` and launch Visual Studio:
-
-```dotnetcli
-dotnet tool restore
-dotnet cake --target=VS
-```
-
-There is also a `VSCode` target for launching Visual Studio Code.
-
-```dotnetcli
-dotnet tool restore
-dotnet cake --target=VSCode
-```
-
-#### Testing branch against your project
-`--sln=<Path to SLN>`
-- This will pack .NET and then open a VS instance using the local pack. This is useful if you want to check to see if the changes in a branch will address your particular issues. Pack only runs the first time so you will need to explicitly add the `--pack` flag if you make changes and need to repack.
-
-```dotnetcli
-dotnet tool restore
-dotnet cake --sln="<download_directory>\MauiApp2\MauiApp2.sln" --target=VS
-```
-
-#### Pack
-`--pack`
-- This creates .NET MAUI packs inside the local dotnet install. This lets you use the CLI commands with the local dotnet to create/deploy with any changes that have been made on that branch (including template changes).
-
-```dotnetcli
-dotnet tool restore
-dotnet cake --target=VS --pack --sln="<download_directory>\MauiApp2\MauiApp2.sln"
-```
-
-Create new .NET MAUI app using your new packs
-```dotnetcli
-dotnet tool restore
-dotnet cake --pack
-mkdir MyMauiApp
-cd MyMauiApp
-..\bin\dotnet\dotnet new maui
-..\bin\dotnet\dotnet build -t:Run -f net[current_sdk_version]-android
-```
-
-You can also run commands individually:
-```dotnetcli
-# install local tools required to build (cake, pwsh, etc..)
-dotnet tool restore
-# Provision .NET SDK in bin\dotnet
-dotnet build src\DotNet\DotNet.csproj
-# Builds Maui MSBuild tasks
-.\bin\dotnet\dotnet build Microsoft.Maui.BuildTasks.slnf
-# Builds the rest of Maui
-.\bin\dotnet\dotnet build Microsoft.Maui.sln
-# Launch Visual Studio
-dotnet cake --target=VS
-```
-
-## Debugging MSBuild Tasks using VS/VSCode
-
-One thing that is very useful is the ability to debug your Tasks while
-they are being run on a build process. This is possible thanks to the
-`MSBUILDDEBUGONSTART` environment variable. When set to `2` this will
-force MSBuild to wait for a debugger connection before continuing.
-You will see the following prompt.
```

Comment:
Some of this other stuff seems generally useful, and we'd lose it. Should we make a "development tips" document? Similar to:

* https://github.com/dotnet/android/blob/main/Documentation/workflow/DevelopmentTips.md

Reply:
alright, added a DevelopmentTips so that knowledge isn't lost :-)